ft-sdk/src/email.rs

```diff
@@ -8,34 +8,45 @@ pub enum EmailError {
 /// requests get picked up by the email worker
 ///
 /// # Arguments
+/// * `from` - (name, email)
 /// * `to` - (name, email)
 /// * `subject` - email subject
 /// * `body` - email body
 /// * `mkind` - mail kind, used for logical logging purposes
+/// * `cc`, `bcc` - comma separated email with names. E.g: <test1@gmail.com>, Bob <test2@ocr-inc.com>
 pub fn send_email(
+    conn: &mut ft_sdk::Connection,
+    from: (&str, &str),
     to: (&str, &str),
     subject: &str,
-    conn: &mut ft_sdk::Connection,
-    // TODO: add support for text emails
-    html_body: &str,
+    body_html: &str,
+    body_text: &str,
+    reply_to: Option<&str>,
```

Comment:
If `from` and `to` are `(&str, &str)`, `reply_to` should be too.

Also to should be a `Vec(&str, &str)`? And should we add `cc`?

Reply:
done in: https://github.com/FifthTry/ft-sdk/pull/26/commits/b937ced1cddd0b49c5239c869200c22a123deb63